Annotato/Annotato/ViewModels/Document/DocumentAnnotationViewModel.swift

```diff
@@ -1,15 +1,32 @@
 import CoreGraphics
+import PDFKit
 
 class DocumentAnnotationViewModel {
-    private(set) var center: CGPoint
     private(set) var parts: [DocumentAnnotationPartViewModel]
     private(set) var width: Double
 
-    init(center: CGPoint, width: Double, parts: [DocumentAnnotationPartViewModel]) {
-        self.center = center
+    unowned var associatedDocumentPdfViewModel: DocumentPdfViewModel?
+    var associatedDocument: PDFDocument
+    var coordinatesInDocumentSpace: CGPoint
+    var associatedPage: PDFPage
+    var coordinatesInPageSpace: CGPoint
+
+    init(
+        associatedDocumentPdfViewModel: DocumentPdfViewModel,
+        coordinatesInDocumentSpace: CGPoint,
```

Comment:
Are these coordinates supposed to be the origin? center? top left corner? bottom right corner? Naming seems a little unclear :<

Reply:
I changed it to origin